Ghidra/Features/Decompiler/src/decompile/cpp/jumptable.cc

```diff
@@ -510,7 +510,7 @@ uintb JumpBasic::backup2Switch(Funcdata *fd,uintb output,Varnode *outvn,Varnode
   return output;
 }
 
-/// If the Varnode has a restricted range due to masking via INT_AND, the maximum value of this range is returned.
+/// If the Varnode has a restricted range due to masking via INT_AND or by INT_REM with a constnant, the maximum value of this range is returned.
```

Comment:
`constant`

Reply:
Thanks for spotting the typo! It's fixed now.